tfhe/benches/shortint/bench.rs

```diff
@@ -304,6 +340,47 @@ fn programmable_bootstrapping(c: &mut Criterion) {
     bench_group.finish();
 }
 
+fn server_key_from_compressed_key_bench(c: &mut Criterion, params_set: BenchParamsSet) {
+    let mut bench_group = c.benchmark_group("uncompress_key");
+    bench_group
+        .sample_size(10)
+        .measurement_time(std::time::Duration::from_secs(60));
+
+    for param in benchmark_parameters(params_set).iter() {
+        let keys = KEY_CACHE.get_from_param(*param);
+        let sks_compressed = CompressedServerKey::new(&keys.client_key());
+
+        let bench_id = format!("shortint::uncompress_key::{}", param.name());
+
+        bench_group.bench_function(&bench_id, |b| {
+            let clone_compressed_key = || {
+                let sks_cloned = sks_compressed.clone();
+                sks_cloned
+            };
```

Comment:
let clone_compressed_key = || sks_compressed.clone(); should be enough I think (clippy will likely yell at you)

Reply:
Ok but how I get my hand on the value? `clone_compressed_key` is a closure not the value returned from the cloning.

Reply:
clone_compressed_key()

one liner closures have the right to omit curly braces